utilities/tools/Test-ModuleLocally.ps1

```diff
@@ -139,7 +138,7 @@ function Test-ModuleLocally {
     begin {
         $repoRootPath = (Get-Item $PSScriptRoot).Parent.Parent
         $ModuleName = Split-Path (Split-Path $TemplateFilePath -Parent) -Leaf
-        Write-Verbose "Running local tests for [$ModuleName]"
+        Write-Verbose "Running Local Tests for $($ModuleName)"
```

Comment:
Please undo. Not only are the extra brackets not required, but in my subjektiv opinion, the original brackets [ ]  make it easier to see which part of an output is the variable one

Reply:
Not sure why this came out as a change as I didn't change it
![image](https://user-images.githubusercontent.com/28486158/184266217-45ed768d-a793-4def-be75-c41a8c6ef29a.png)
But I updated it regardless

Reply:
It may have been a resolved conflict as I believe this line was changed in main recently. Anyhow, thanks :)